Cellular heterogeneity in biological samples is a key factor that determines disease progression, but also influences biomedical analysis of samples and patient classification. At the molecular level, the cellular composition of tissues is difficult to assess and quantify, as it is hidden within the bulk molecular profiles of samples (average profile of millions of cells), with all cells present in the tissue contributing to the recorded signal. Despite great promise, conventional computational approaches to quantifying cellular heterogeneity from mixtures of cells have encountered difficulties in providing robust and biologically relevant estimates.
Here, our focus will be on reference-based approaches, which are gaining increasing popularity. While each method presents its own set of advantages and limitations, all are inherently constrained by the quality of the reference data employed. 

**The objective of the HADACA3 challenge will be to enhance existing cell-type deconvolution models by integrating multimodal datasets as reference data.**

**Phase 1:** 

		
**Phase 2:** Estimate cell type heterogeneity from prancreatic matching bulk methylomes and transcriptomes using the following references profiles :

- bulk RNA-seq references of 5 cell types,
- DNAm references of 5 cell types,
- single-cell RNA-seq profiles of 5 cell types

**Phase 3:** 
